src/transformStateWithClones.js

```diff
@@ -6,8 +6,37 @@
  *
  * @return {Object[]}
  */
+
 function transformStateWithClones(state, actions) {
   // write code here
+  const array = [];
+  let stateCopy = { ...state };
+  let pushedCopy = {};
```

Comment:
You could do it with just one constant for a state copy
```suggestion
  const stateCopy = { ...state };
```